cmd/root.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const ccJSONLocation = "coverage/codeclimate.json"
```

Comment:
Since we're here... ;) How about (assuming that `cc` bit is needed or conventional), `ccDefaultCoveragePath`? There might be many "locations" and "JSON" things in this code base some day.